app/controllers/user_orders_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UserOrdersController < ApplicationController
+  def index
+    @user = User.find(session[:user_id])
```

Comment:
Do we need this? I think current_user takes care of finding who is currently login